tsconfig.json

```diff
@@ -16,7 +16,7 @@
       "@/types/*": ["packages/types/*"]
     }
   },
-  "include": ["src/**/*", "grpc_codegen/**/*", "tests"],
```

Comment:
The root folder tsconfig.json doesn’t need `"src/**/*"`. I forgot to remove
 it after intialization.